deepecho/__init__.py

```diff
@@ -1,16 +1,16 @@
 """Top-level package for DeepEcho."""
 
-__author__ = 'DataCebo, Inc.'
-__email__ = 'info@sdv.dev'
-__version__ = '0.5.1.dev0'
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+__author__ = "DataCebo, Inc."
```

Comment:
why switch to double quotes?

Reply:
Yeah I was just wondering if there was a universal standard. We've been using single quotes for all our libraries

Reply:
pep doesn't dictate a standard so I am fine with using single quotes.